src/styles/typography.ts

```diff
@@ -32,7 +32,7 @@ const subHeadingsFont = localFont({
 const bodyFont = localFont({
   src: [
     {
-      path: "./fonts/appleII.ttf",
+      path: "./fonts/Apple II Pro.otf",
```

Comment:
Please rename that font file to use dashes instead of spaces. Spaces can really mess up paths and it's better to not have them in the first place